packages/langium/src/generate/node-joiner.ts

```diff
@@ -167,7 +167,7 @@ export function joinTracedToNode<T extends AstNode>(astNode: T, property?: Prope
  *  arguments as expected by {@link joinToNode}, i.e. an `iterable`, a function `toGenerated`
  *  converting each element into a `Generated`, as well as some `options`.
  *
- * That function than joins the elements of `iterable` by delegating to {@link joinToNode}.
+ * That function then joins the elements of `iterable` by delegating to {@link joinToNode}.
```

Comment:
Suggestion: The signature before this change also needs to be adjusted.

Reply:
Right, seems I forgot it earlier